examples/src/index.js

```diff
@@ -40,7 +40,7 @@ import spacer from 'ory-editor-plugins-spacer'
 import 'ory-editor-plugins-spacer/lib/index.css'
 
 // The image plugin
-import image from 'ory-editor-plugins-image'
+import { imagePlugin } from 'ory-editor-plugins-image'
```

Comment:
ah cool, I was thinking you might be on holidays or something. afk now but I think this is not a breaking change. the only problem is the plugins now are exported as object instead of functions (the default export). that is generally not s good idea because it makes it impossible to pass in config. what I did here is i left the defaul export as an object (for backwards compatibility) and added the named export as a function so that you can configure it yourself. we could make a not of this in docs. but i think for the future, it would make sense to convert all plugins to be functions that return objects.

Reply:
Ahh, that makes sense - in that case no need for documenting this as a BC